core/include/usBundle.h

```diff
@@ -244,6 +245,16 @@ class US_Core_EXPORT Bundle : public std::enable_shared_from_this<Bundle>
   BundleVersion GetVersion() const;
 
   /**
+   * Returns this bundle's Manifest properties as key/value pairs.
+   *
+   * @return A map containing this bundle's Manifest properties as 
+   *         key/value pairs.
+   *
+   * @sa \ref MicroServices_BundleProperties
+   */
+  std::map<std::string, Any> GetProperties() const;
```

Comment:
We could probably rename the `ServiceProperties` typedef (a std::unordered_map) to `Dictionary` and use this consistently.